compass/landice/tests/slm/circ_icesheet/circ_icesheet.cfg

```diff
@@ -0,0 +1,79 @@
+# Config options for slm circ_icesheet test case
+[circ_icesheet]
+
+# The size of the domain in meters in the x and y directions for the planar mesh
+lx = 6000000.0
+ly = 6000000.0
+
+# list of MALI mesh resolutions in kilometers delimited by ',' without space
+# i.e. The distance between adjacent cell centers.
+mali_res = 20
+
+# Ice shape type ('cylinder', and for dome, 'dome-halfar' or 'dome-cism')
+# r0 and h0: initial radius and height of the cylinder ice in meters
+ice_type = cylinder
+r0 = 2000000.0
+h0 = 3000.0
+
+# 'True' if manually want to set bedTopography elevation
+# bed topography elevation in meters (provide a positive/negative float value
+# for grounded above land/marine-based ice)
+set_topo_elev = True
+topo_elev = -1500.0
+
+# Whether to center the circular ice in the center of the cell that is closest to the
+# center of the domain
+put_origin_on_a_cell = False
+
+# Config options for surface mass balance forcing
+[smb_forcing]
+
+# Start, end and interval years for SMB forcing
+start_year = 2015
+end_year = 2116
+dt_year = 1
+
+# Direction in which SMB is applied ('horizontal' or 'vertical')
+# and amount of ice to melt
+direction = horizontal
+# Change in radius in meters; used when 'direction == horizontal'
+drdt = -2000.0
+# Change in height in meters;sed when 'direction == vertical'
+dhdt = -20.0
+
+# config options for the sea-level model
+[slm]
+# True if MALI-SLM are coupled
+coupling = True
+
+# List of the number of Gauss-Legendre points in latitude
+# delimited by ',' without space
+slm_nglv = 512
+
+# Max spherical harmonics degree and order
+# i.e. SLM resolution of the SLM
+slm_res = 512
+
+# mapping method between the MALI and SLM grids
+mapping_method_mali_to_slm = conserve
+mapping_method_slm_to_mali = bilinear
+
+# ratio of MALI-SLM coupling interval and MALI output interval in integer years
+time_stride = 1
```

Comment:
I see this is used in visualize.py but I don't see it used anywhere else.  Is this a value that the user needs to be set to be consistent with what is in the namelist and streams files?  If so, let's talk about what would be needed to calculate this programmatically and eliminate the option.